docs/man/containerd-config.toml.5.md

```diff
@@ -200,6 +204,7 @@ imports = ["/etc/containerd/runtime_*.toml", "./debug.toml"]
     platforms = ["linux/amd64"]
     sched_core = true
   [plugins."io.containerd.service.v1.tasks-service"]
+    blockio_config_file = "/etc/blockio-config.yaml"
```

Comment:
did you want the default to be `""` or `"/etc/blockio-config.yaml"`

FYI you can get the default by running "containerd config default"..

Reply:
Fixed the defaults to match real defaults in both blockio_config_file and rdt_config_file (the next line).